### ISSUE TITLE
Add Perplexideez

### ISSUE THREAD

**Issue by nullnuller** [av/harbor#81]
https://github.com/brunostjohn/perplexideez

A multi-user perplexity clone that supports both ollama and docker. Thought it might be interesting.

**Comment by av** [comment]
Yup, thanks! I've seen it on Reddit and also noted it down for possible integration

**Comment by av** [comment]
I tried, but [failed](https://github.com/av/harbor/blob/main/compose.perplexideez.yml). We'll have to wait for some changes on the upstream to make it run on `localhost` without issues

**Comment by av** [comment]
It looks like Perplexideez development completely stopped 5 months ago, I'm open to contributions with this service, but won't work on it myself - hence closing the issue
